taxcalc/functions.py

```diff
@@ -293,22 +298,21 @@ def ItemDed(_posagi, e17500, e18400, e18500, e18800, e18900,
               c19700 + c20500 + c20800 + e21000 + e21010)
 
     # Limitations on deductions excluding medical, charity etc
-    _phase2_i = ID_ps[MARS - 1]
-
     _nonlimited = c17000 + c20500 + e19570 + e21010 + e20900
     _phase2_i = ID_ps[MARS - 1]
     _limitratio = _phase2_i / _sep
 
+    # Itemized deductions amount after limitation if any
+    c04470 = c21060
+
     if c21060 > _nonlimited and c00100 > _limitratio:
         dedmin = ID_crt * (c21060 - _nonlimited)
         dedpho = ID_prt * max(0, _posagi - _limitratio)
         c21040 = min(dedmin, dedpho)
+        c04470 = c21060 - c21040
```

Comment:
Form 1040 Schedule A: Line 29. Corresponding to case:
"Yes. Your deduction may be limited. See the Itemized Deductions Worksheet in the instructions to figure the amount to enter."